tutorials/position-velocity-diagrams/PVDiagramPlotting.ipynb

```diff
@@ -0,0 +1,553 @@
+{
```

Comment:
<p>Describe a bit more what the path is. E.g., this is the path through the data from which we'll calculate the velocities.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/503/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Let's plot one channel (slice) of the cube so we can see the image and decide where we want to draw the path where we'll calculate the positions and velocities. In this section, we'll use pixel units, we'll convert to sky coordinates in the next section.</p>

Reply:
<p>rephrased/restructured</p>